README.md

```diff
@@ -13,18 +13,23 @@
 [![GoDoc](https://godoc.org/github.com/getsentry/sentry-go?status.svg)](https://godoc.org/github.com/getsentry/sentry-go)
 [![go.dev](https://img.shields.io/badge/go.dev-pkg-007d9c.svg?style=flat)](https://pkg.go.dev/github.com/getsentry/sentry-go)
 
-`sentry-go` provides a Sentry client implementation for the Go programming language. This is the next line of the Go SDK for [Sentry](https://sentry.io/), intended to replace the `raven-go` package.
+`sentry-go` provides a Sentry client implementation for the Go programming
+language. This is the next line of the Go SDK for [Sentry](https://sentry.io/),
+intended to replace the `raven-go` package.
 
 > Looking for the old `raven-go` SDK documentation? See the Legacy client section [here](https://docs.sentry.io/clients/go/).
 > If you want to start using sentry-go instead, check out the [migration guide](https://docs.sentry.io/platforms/go/migration/).
 
 ## Requirements
 
-We verify this package against N-2 recent versions of Go compiler. As of September 2019, those versions are:
```

Comment:
This is to avoid constant manual maintenance: we don't need to list out version numbers explicitly and keep them up-to-date every ~6 months.